README_Korean.md

```diff
@@ -0,0 +1,215 @@
+
+RxAlamofire
+===
+
+RxAlamofire는 Swift [Alamofire](https://github.com/Alamofire/Alamofire)의 우아한 HTTP 네트워킹을 중심으로 한 RxSwift 기반의 라이브러리입니다.
+
+[![CircleCI](https://img.shields.io/circleci/project/github/RxSwiftCommunity/RxAlamofire/master.svg)](https://circleci.com/gh/RxSwiftCommunity/RxAlamofire/tree/master)
+[![Version](https://img.shields.io/cocoapods/v/RxAlamofire.svg?style=flat)](http://cocoapods.org/pods/RxAlamofire)
+[![License](https://img.shields.io/cocoapods/l/RxAlamofire.svg?style=flat)](http://cocoapods.org/pods/RxAlamofire)
+[![Platform](https://img.shields.io/cocoapods/p/RxAlamofire.svg?style=flat)](http://cocoapods.org/pods/RxAlamofire)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+
+## 시작하기
+
+Alamofire라이브러리에 RxSwift를 적용하는 것은 네트워크 요청을 더욱 더 유연하고 멋있게 수행할 수 있도록 해줍니다.
+Alamofire는 가장 강력한 프레임워크이며 RxSwift를 사용함으로써 더욱 더 단순하고 효과적인 방법으로 응답을 구성할 수 있게 됩니다.
+
+기본적인 사용법은 다음과 같습니다.
+(considering a simple currency converter):
+
+```swift
+let formatter = NSNumberFormatter()
+formatter.numberStyle = .currencyStyle
+formatter.currencyCode = "USD"
+if let fromValue = NSNumberFormatter().numberFromString(self.fromTextField.text!) {
+
+RxAlamofire.requestJSON(.get, sourceStringURL)
+.debug()
```

Comment:
We need to keep the original indentations.

Reply:
I modified!